index.html

```diff
@@ -44,29 +44,29 @@ <h1 class="greet">
         <ul class="social-icon">
           <li class="twitter-handle">
             <a href="https://twitter.com/RishiMi31357764"
-              ><img src="assets/Vector-3.png" ></a>
+              ><img src="assets/Vector-3.png" alt="twitter" ></a>
           </li>
           <li class="linkedin-handle">
             <a href="https://www.linkedin.com/in/rishi-mishra-756718257/"
-              ><img src="assets/Linkedin icon.png"></a>
+              ><img src="assets/Linkedin icon.png" alt="linked in"></a>
           </li>
-          <li><a href="#"><img src="assets/Vector-1.png"></a></li>
+          <li><a href="#"><img src="assets/Vector-1.png" alt="medium"></a></li>
           <li class="github-handle">
             <a href="https://github.com/Rishi-Mishra0704"
-              ><img src="assets/Vector-2.png">
+              ><img src="assets/Vector-2.png" alt="github">
             </a>
           </li>
-          <li><a href="#"><img src="assets/Vector.png"></a></li>    
+          <li><a href="#"><img src="assets/Vector.png" alt="angelist"></a></li>    
         </ul>
       </section></div>
       <section class="work">
         <div class="project-container" id="project_1">
           <div class="work-img">
-            <img src="assets/Snapshoot Portfolio.png">
+            <img src="assets/Snapshoot Portfolio.png" alt="work-card">
           </div>
           <div class="project-title">Tonic</div>
           <div class="some-list">
-            <img src="assets/Frame 2.png">
+            <img src="assets/Frame 2.png" alt="">
```

Comment:
[OPTIONAL] Hey 👋🏽 I noticed here that you're using the image of this section instead of using the texts, I'm not asking to change it because that would require redoing a good part of the design 🎨 but it can be a problem when people want to navigate through your web 🕸️ page using tab and keyboard. ⌨️